source/stylesheets/posts.css.scss

```diff
@@ -1,593 +1,131 @@
-            column-count: 0;
-        }
-
-        @media (min-width: 900px) {
-            column-count: 2;
-            column-gap: 4em;
-        }
-
-        .typography {
-            column-span: all;
-            padding-top: 3%;
-            text-align: left;
-
-            @media (min-width: 300px) {
-                font-size: 1.15em;
-                text-align: center;
-            }
-
-            @media (min-width: 768px) {
-                font-size: 1.5em;
-                text-align: left;
-            }
-
-            @media (min-width: 900px) {
-                font-size: 1.75em;
-                text-align: left;
-            }
-        }
-
-        h2 {
-            font-size: 1.5em;
-        }
-
-        h3 {
-            @media (min-width: 300px) {
-                font-size: 1em;
-                padding-left: 5%;
-                text-align: left;
-            }
-
-            @media (min-width: 768px) {
-                font-size: 1.25em;
-                padding-left: 0;
-                text-align: left;
-            }
-
-            @media (min-width: 900px) {
-                font-size: 1.25em;
-                text-align: left;
-            }
-        }
-
-
-        h4 {
-            padding-top: 0;
-        }
-
-        p {
-            padding-top: 10px;
-
-            @media (min-width: 768px) {
-                font-size: 1em;
-                text-align: left;
-            }
-        }
-
-        ul {
-            list-style-type: disc;
-
-            li {
-                padding-top: 8px;
-            }
-        }
-    }
-}
-
-.blogposts__space--above {
-    padding-top: 5%;
-}
-
-.blogposts__ul {
-    padding-left: 0;
-
-    li {
-        list-style: disc;
-        font-size: 14px;
-        line-height: 200%;
-        padding-left: 1em;
-    }
-}
-
-.blogposts__ul--none {
-    list-style: none;
-
-    li {
-        list-style: none;
-        padding-left: 1em;
-    }
-}
-
-.blogposts__ul--indent {
-    padding-left: 1.25em;
+main {
+  align-content: center;
+  background: $c-mandys-pink;
+  padding-bottom: 20%;
+  padding-left: 15%;
+  padding-right: 15%;
+  width: 100%;
 
-    @media (min-width: 300px) {
-        font-size: .85em;
-        padding-left: 5%;
-        padding-top: 20px;
-        text-align: left;
-    }
-
-    @media (min-width: 768px) {
-        font-size: 1em;
-        padding-left: 0;
-        padding-top: 20px;
-        text-align: left;
-    }
-
-    @media (min-width: 900px) {
-        font-size: 1em;
-        padding-left: 0;
-        padding-top: 0;
-        text-align: left;
-    }
-
-    li {
-        list-style: disc;
-        font-size: 14px;
-        line-height: 200%;
-        margin-left: 1.5em;
+  h2 {
+    column-span: all;
+    text-align: left;
+  }
 
-        @media (min-width: 300px) {
-            font-size: 1em;
-            line-height: 150%;
-        }
-    }
+  .img {
+    display: block;
+    padding-left: 20%;
+    margin-left: auto;
+    margin-right: auto;
```

Comment:
This is another spot where you can combine multiple properties into one. Margin-left and margin-right can just be margin. You can rewrite this "margin: 0 auto" and the auto will be applied to margin-left and margin-right, and 0 will be applied to margin-top and margin-bottom. Just make sure that you want margin-top and margin-bottom to be 0 first (it can be anything you want, like 1em.)